lib/kitchen/provisioner/install.erb

```diff
@@ -72,8 +72,72 @@ else
   exit 2
 fi
 
-#{install_chef}
+echo "-----> Installing other dependencies..."
+YUM_CMD="$(which yum)"
```

Comment:
the use of `which` should be avoided. Info on why and what to use instead on https://stackoverflow.com/a/677212